docs/UserGuide.md

```diff
@@ -413,6 +413,23 @@ Example:
 
 ![Appointments Between Wireframe](images/FreeBetween.png)
 
+### Finding Appointment by Name or Person Name : `findappt`
+
+Finds appointments with name or Person name that contains the given keyword.
+
+Format: `findappt KEYWORD [MORE_KEYWORDS] [by/person]`
```

Comment:
Small edit, this should be `[by/(person OR name)]`

Reply:
Fixed